### PR TITLE
An asdf tag for Linear1D

### PR DESCRIPTION
**Pull request by Cadair**
needs https://github.com/spacetelescope/asdf-standard/pull/159

### CODE REVIEW

File: astropy/io/misc/asdf/tags/transform/polynomial.py [hunk 1]
```diff
@@ -125,3 +125,33 @@ def assert_equal(cls, a, b):
...
+            'intercept': _parameter_to_value(model.intercept),
+        }
+        return yamlutil.custom_tree_to_tagged_tree(node, ctx)
+
+    @classmethod
+    def assert_equal(cls, a, b):
+        # TODO: If models become comparable themselves, remove this.
+        TransformType.assert_equal(a, b)
+        assert (isinstance(a, modeling.models.Scale) and
+                isinstance(b, modeling.models.Scale))
```
Comment (nden) [inline astropy/io/misc/asdf/tags/transform/polynomial.py]: These two need to be `Linear1D`.

**Comment by drdavella** [comment]
image

**Comment by Cadair** [comment]
I think the coveralls tests is lying. The tests are [being skipped](https://travis-ci.org/astropy/astropy/jobs/364157346#L4449), there is a `skipif` at the top of the test file.

**Comment by drdavella** [comment]
Thanks @Cadair!

**Comment by pllim** [comment]
@drdavella , I am saving that pic for future use. :smile:

**Pull request merged**